Update messaging-system.md (#181)
Unity-Technologies/com.unity.multiplayer.docs: docs/advanced-topics/messaging-system.md

---
id: messaging-system
title: Messaging System and RPCs

> description: An introdution to the messaging system in MLAPI covering RPC's and Custom Messages

Reviewer: Just wondering why the 0.1.0 docs for messaging-system don't use this description and content? I'm pretty sure that this should apply for both develop and the 0.1.0 version.

Author: I haven't put it in the develop yet, we are updating pages for accessibility as we touch them. so the four files I have touched here have the changes added. most of the other pages don't. Once this is approved then I will make the changes to develop as well. I will chat with Lori to see the best way to do this.

Reviewer 2: You need to update both at the same time when updating any content. If it affects both, update in both. :)

Author: don e

> description: An introdution to the messaging system in MLAPI covering RPC's and Custom Messages

Reviewer 2: ```suggestion
description: An introduction to the messaging system in Unity MLAPI, including RPC's and Custom Messages.
```

---

> The MLAPI has two parts to it's messaging system. [RPC messages](about-rpc.md) and [Custom Messages](custom-messages.md). Both types have sub types that change their behaviour, functionality and performance.

Reviewer 2: ```suggestion
The MLAPI has two parts to its messaging system: [RPC messages](about-rpc.md) and [Custom Messages](custom-messages.md). Both types have sub-types that change their behaviour, functionality, and performance.
```

A method turned into an RPC is no longer a regular method, it will have its own implications on direct calls and in the network pipeline. See [Execution Table](message-system/execution-table.md).

## Network Update Loop

> See [Network Update Loop](network-update-loop-system/index.md) for information on the update infrastructure.

Reviewer 2: You removed this in /docs (develop) and kept it in 0.1.0...should this also be kept?

## Network Update Loop

See [Network Update Loop](network-update-loop-system/index.md) for information on the update infrastructure.

## Custom Messages

> See [Custom Messages](custom-messages.md) for information on tCustom Messages.

Reviewer: ```suggestion
See [Custom Messages](custom-messages.md) for information on Custom Messages.
```

Author: Both changes now in palce
